server/models/creature.js

```diff
@@ -1,222 +1,222 @@
-var mongoose = require('mongoose');
-var SharingSchema = require('./sharingSchema');
-
-//Flavor
-var CreatureFlavor = new mongoose.Schema(
-    {
-        faction: {
-            type: String,
-            default: ""
-        },
-        environment: {
-            type: String,
-            default: ""
-        },
-        description: {
-            type: String,
-            default: ""
-        },
-        nameIsProper: {
-            type: Boolean,
-            default: false
-        },
-        imageUrl: {
-            type: String,
-            default: ""
-        }
-    },
-    {
-        _id : false
-    });
-
-//6 core ability sores
```

Comment:
I changed it to be scores, no other change to this file (I don't have OCD, really)